opm/material/densead/Evaluation.hpp

```diff
@@ -47,26 +48,26 @@ namespace DenseAd {
  * \brief Represents a function evaluation and its derivatives w.r.t. a fixed set of
  *        variables.
  */
-template <class ValueT, unsigned numVars>
```

Comment:
if I remember correctly, these warnings do not occur with `-Wall` but only with `-Wextra` -- or on clang also with `-Weverything`. I don't see them as a problem either (originally it was `int` IIRC), but @atgeirr probably begs to differ ;)